Data/Sys/GameSettings/WD9.ini

```diff
@@ -14,4 +14,4 @@ CPUThread = False
 # Add action replay cheats here.
 
 [Video_Hacks]
-EFBToTextureEnable = False
```

Comment:
Is the removal of EFB2RAM here correct?

Reply:
Yes, in older Dolphin version EFB to texture only creates visual clitches but has been fixed since at least 5.0.